packages/reactive-element/src/test/decorators/property_test.ts

```diff
@@ -264,7 +264,13 @@ suite('@property', () => {
         return {foo: {type: Number, reflect: true}};
       }
 
-      foo = 1;
+      declare foo: number;
```

Comment:
Have we now lost all coverage of declared class fields in TS with `useDefineForClassFields=false`